### DISCUSSION TITLE
[Naive Receiver] Hints to solve challenge?

### DISCUSSION THREAD

**Discussion by Ed-Marcavage** [tinchoabbate/damn-vulnerable-defi#60] [Help / Q&A / Troubleshooting] [answered]
Hello everyone, I am stuck on Naive Receiver and would appreciate any hints, resources, or advice to help progress. 

Currently, I drained the FlashLoanReceiver contract's 10 ETH, but I'm stuck on how to get the ETH out of the NaiveReceiverPool; any leads would be appreciated.

**Comment by tinchoabbate** [comment, marked as answer]
Hi @Ed-Marcavage, have you checked the meta-transactions implementation in the BasicForwarder? May be worth taking a deep look there.
